packages/stark-ui/src/modules/route-search/components/route-search.component.ts

```diff
@@ -0,0 +1,199 @@
+import {Component, Inject, Input, OnInit, ViewEncapsulation} from "@angular/core";
+import { RouteEntry } from "../entities/route-entry";
+import { Observable } from "rxjs/internal/Observable";
+import { FormControl } from "@angular/forms";
+import {map, startWith} from "rxjs/operators";
+import { STARK_ROUTING_SERVICE, StarkRoutingService, STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { UIRouter } from "@uirouter/core";
+import {StarkMenuConfig, StarkMenuGroup, StarkMenuSection} from "@nationalbankbelgium/stark-ui";
+export type StarkRouteSearchDirection = "left" | "right" | "";
```

Comment:
We should set a default direction instead of using `""`. Can you remove the empty string from these types?